File: Revisao/incremental3/Transacao.java

class Transacao {
    private float valor;
    private String descricao;

    public Transacao(float valor, String descricao) {
        this.valor = valor;
        this.descricao = descricao;
    }

    public float getValor() {
        return valor;
    }

    public String getDescricao() {
        return descricao;
    }

    public void detalhesDaTransacao() {
        System.out.println("Valor: " + valor + ", Descrição: " + descricao);
    }
}

File: Revisao/python/transacao.py

from datetime import datetime

class Transacao:
    def __init__(self, data, valor, descricao, tipo):
        self.data = data
        self.valor = valor
        self.descricao = descricao
        self.tipo = tipo
    def detalhes_da_transacao(self):
        print(f"Data: {self.data}, valor: R${self.valor}, descrição: {self.descricao}, tipo: {self.tipo}") 
        
